domains/cgprograms.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "CompuGenius Programs creates programs and games to better the computer world.",
+  "repo": "https://github.com/compugenius",
```

Comment:
This should be the GitHub repo for the website, if you don't have one, then remove this line.